.github/workflows/prettier.yml

```
@@ -34,3 +34,20 @@ jobs:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      - name: Setup
+        uses: actions/setup-node@v4
+        with:
+          node-version: 18
+
++      - name: Install dependencies
++        run: npm ci
+
+      - name: Lint
```

actionlint (1.7.9)

could not parse as YAML: mapping values are not allowed in this context

(syntax-check)